kroxylicious-test-tools/src/main/java/io/kroxylicious/test/tester/KroxyliciousTesterBuilder.java

```diff
@@ -0,0 +1,100 @@
+/*
+ * Copyright Kroxylicious Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+package io.kroxylicious.test.tester;
+
+import java.util.function.Function;
+
+import io.kroxylicious.proxy.config.Configuration;
+import io.kroxylicious.proxy.config.ConfigurationBuilder;
+
+import edu.umd.cs.findbugs.annotations.Nullable;
+
+/**
+ * Kroxylicious strives to offer high quality testing tools which allow filter authors to focus on their business logic.
+ * However, as Kroxylicious is very configurable the so is the Tester. Most of the time users do not care about a lot of the configuration options
+ * and thus a builder, which can provide sensible defaults, provides the simplest API to configure things.
+ */
+public class KroxyliciousTesterBuilder {
```

Comment:
I think your intent is the io.kroxylicious.test.tester.KroxyliciousTesters#newBuilder is the api for creating a builder. so should this class have a default ctor to prevent it being instantiated from elsewhere?

Reply:
TBH the builder in this PR is the minimum thing to get the topic manager moving. My intention was to flesh it out (by expanding into building the mock tester as well in https://github.com/kroxylicious/kroxylicious/pull/661) I think thats going to change a lot of the details.

Does that work as an approach for you?